network/protocol.go

```diff
@@ -32,6 +32,18 @@ import (
 	"github.com/ethersphere/swarm/state"
 )
 
+var (
+	capabilitiesRetrieve      = 0
```

Comment:
are you spacing these to leave space for future capabilities?

Reply:
Yes